snap/local/hooks/cmd/install/install.go

```diff
@@ -0,0 +1,392 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	hooks "github.com/canonical/edgex-snap-hooks/v2"
+)
+
+var cli *hooks.CtlCli = hooks.NewSnapCtl()
+
+const proxyRouteCfg = "env.security-proxy-setup.add-proxy-route"
+const secretStoreAddTokensCfg = "env.security-secretstore-setup.add-secretstore-tokens"
+const secretStoreAddKnownSecretsCfg = "env.security-secretstore-setup.add-known-secrets"
+const consulAddRegistryACLRolesCfg = "env.security-bootstrapper.add-registry-acl-roles"
```

Comment:
absolutely right. I was looking at the "services" array in install.go, rather than "Services" in const.go - and that has "security-bootstrapper" as well as "security-proxy-setup" and "security-secretstore-setup". I'll change all of this to use the correct name (minus "setup") and then we need to add "security-bootstrapper" as a service in const.go. We could then just as well add add-known-secrets and add-registry-acl-roles at the same time to const.go